src/main/java/com/auth0/json/mgmt/jobs/Job.java

```diff
@@ -0,0 +1,56 @@
+package com.auth0.json.mgmt.jobs;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Class that represents an Auth0 Job object. Related to the {@link com.auth0.client.mgmt.JobsEntity} entity.
+ */
+@SuppressWarnings({"unused", "WeakerAccess"})
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Job {
+
+    @JsonProperty("status")
+    private String status;
+    @JsonProperty("type")
+    private String type;
+    @JsonProperty("created_at")
+    private String createdAt;
+    @JsonProperty("id")
+    private String id;
+
+    @JsonCreator
+    public Job(@JsonProperty("status") String status, @JsonProperty("type") String type, @JsonProperty("id") String id) {
+        this.status = status;
+        this.type = type;
+        this.id = id;
+    }
+
+    @JsonProperty("status")
+    public String getStatus() {
+        return status;
+    }
+
+    @JsonProperty("type")
+    public String getType() {
+        return type;
+    }
+
+    @JsonProperty("created_at")
+    public String getCreatedAt() {
+        return createdAt;
+    }
+
+    @JsonProperty("id")
+    public String getId() {
+        return id;
+    }
+
+    @JsonProperty("created_at")
+    public void setCreatedAt(String createdAt) {
```

Comment:
I don't see any sample values for this `created_at` field in the api docs. But probably would be better to just use a Date and make Jackson convert between the types. I think the API already makes use of the ISO-8601 format. See the User's class created_at method. If this is the case, then you'll also need to update the JSON samples used in the introduced tests.